ciao-cli/identity.go

```diff
@@ -252,36 +252,27 @@ func getTenant(username string, password string, tenantID string) (string, strin
 		return "", "", err
 	}
 
-	if len(projects) == 0 {
+	if len(projects) <= 0 {
```

Comment:
no that I see, but I prefere to use <= 0  than just == 0, I mean it does not add extra complexity, and in the case in the future that function is modified and the  len(instances) becames a paramter that function would be a bit more ready, minimal prepation, not that big improvement  

This PR was to remove the code duplication as main purpose